ui/components/package.json

```diff
@@ -40,7 +40,7 @@
     "@uiw/react-codemirror": "^4.19.1",
     "date-fns": "^2.28.0",
     "date-fns-tz": "^1.3.7",
-    "echarts": "^5.4.1",
+    "echarts": "npm:echarts-nightly@^5.5.0-dev.20230425",
```

Comment:
This nightly build is needed to pull in Julie's ECharts [PR here](https://github.com/apache/echarts/pull/18524), which fixes cases where lines would flicker as they were de-emphasized. I'm hoping the next ECharts minor release comes out soon, but so far I haven't found any bugs and it seems like the `v5.5` branch has had a [decent amount of time](https://www.npmjs.com/package/echarts-nightly?activeTab=versions) to bake so maybe the risk is okay assuming we thoroughly test other use cases internally? 🤷‍♂️

Reply:
I think if we're pointing at a very specific nightly that we've tested against a bunch (as opposed to regularly pulling in the nightly) and keep an eye out for the next release and swap to it when it's ready, that it's probably ok.